### PR TITLE
(grc_conversion) EpiDoc and CTS conversion 

### PR DESCRIPTION
**Pull request by lcerrato**
tlg4091; start of tlg0086 file work #1399

### CODE REVIEW

File: data/tlg0086/tlg009/tlg0086.tlg009.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0086/tlg009/tlg0086.tlg009.perseus-grc2.xml:1]: I know they are commented out but why not just delete:

`
</monogr>
`

Also I'd change the link to the page level of the work (such long prefaces in the Teubners!  :) )
https://archive.org/details/aristotelisethic00aris_1/page/1/mode/1up

Comment (lcerrato) [reply data/tlg0086/tlg009/tlg0086.tlg009.perseus-grc2.xml:1]: So I won't forget them in the next file(s)?

Comment (lcerrato) [reply data/tlg0086/tlg009/tlg0086.tlg009.perseus-grc2.xml:1]: The page link is fixed but not checked in.

Comment (AlisonBabeu) [reply data/tlg0086/tlg009/tlg0086.tlg009.perseus-grc2.xml:1]: Ok, I will approve!

**Pull request merged**